fmriprep/workflows/bold.py

```diff
@@ -381,6 +384,49 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
     return workflow
 
 
+def init_bold_reference_wf(omp_nthreads, bold_file=None, name='bold_reference_wf'):
+    """Generate a reference BOLD image for a series
+
+    This image is the target of head-motion-correction, and the subject of distortion
+    correction and registration to T1w and template spaces.
```

Comment:
The plan was to get the docstring right in whichever PR comes second, since there are going to be conflicts either way.